test/RemoteMirror/Inputs/interop.swift

```diff
@@ -9,10 +9,17 @@ case two
 case three
 }
 
+enum F {
+case one(AnyObject)
+case two
+case three
+}
+
 class C {
   let x = "123"
   let y = 456
   let e = E.two
+  let f = F.three
 }
 
```

Comment:
This could do with additional test cases to ensure that we've covered all of the `RecordKind::` cases for which the new code has been added.

Reply:
Added a pile of extra test cases.